__tests__/index.test.js

```diff
@@ -28,4 +42,12 @@ describe('GET /api/topics', () => {
             expect(topics.length).toBe(3)}
         )
     });
+    test('GET 200, each topic object should have 2 keys, slug & description', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body: {topics}}) =>{
+            expect(Object.keys(topics[0])).toEqual([ 'slug', 'description' ])}
```

Comment:
The test description implies it is checking every element in the array because it uses the word "each". However the test only asserts against the first element in the topics array. 

If only there was an array method with the word "each" in it you could use to make multiple assertions 😄 

Such a test should also include an expected array length assertion in it - so if for any reason an empty array is sent back then that test would not reach the topics.forEach thus giving a false positive.